llvm/test/Transforms/Attributor/nofpclass-implied-by-fcmp.ll

```diff
@@ -2641,8 +2641,8 @@ define float @assume_false_smallest_normal(float %arg) {
 }
 
 define float @clamp_false_nan(float %arg) {
-; CHECK-LABEL: define float @clamp_false_nan(
-; CHECK-SAME: float returned [[ARG:%.*]]) #[[ATTR2]] {
+; CHECK-LABEL: define nofpclass(nan inf nzero sub norm) float @clamp_false_nan(
+; CHECK-SAME: float returned nofpclass(nan inf nzero sub norm) [[ARG:%.*]]) #[[ATTR2]] {
```

Comment:
@jdoerfert this is the case I was looking at in #78507. It makes no sense to infer anything here

Reply:
I know what is wrong, did https://github.com/jdoerfert/llvm-project/tree/fpclass_fixes help?
It is just very intrusive and you probably want to populate the switch.

Reply:
Think it's the first part, not the second

Reply:
This turned out to be the issue fixed in 1a6953a75d3120a4f9196911e6a6b79fcc315553